modules/nf-core/bioawk/bioawk.diff

```diff
@@ -0,0 +1,24 @@
+Changes in module 'nf-core/bioawk'
+--- modules/nf-core/bioawk/main.nf
++++ modules/nf-core/bioawk/main.nf
+@@ -12,7 +12,8 @@
+ 
+     output:
+     tuple val(meta), path("*.gz"), emit: output
+-    path "versions.yml"             , emit: versions
++    tuple val(meta), env(LONGEST), emit: longest
```

Comment:
Just to understand, why do you have the (LONGEST) as `env`? Isnt it just counting the length of contigs?

Reply:
I needed a way to export JUST the value of the longest contig, so I can attach it to the `meta` that can be evaluated later in the pipeline.

So by assigning it to the ENV variable, I can isolate it at the output and then add it to meta with the `map` later on. If that makes sense?

Reply:
Oh, that makes sense now. Thx!